src/SDKs/Batch/Support/FileConventions/Src/AzureBatchFileConventions/AzureBatchFileConventions.csproj

```diff
@@ -8,7 +8,7 @@ by task and job id and the type of output.
     </Description>
     <AssemblyTitle>Microsoft Azure Batch File Conventions</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Batch.Conventions.Files</AssemblyName>
-    <VersionPrefix>3.1.0</VersionPrefix>
+    <VersionPrefix>4.0.0</VersionPrefix>
```

Comment:
Is there a reason you would like to bump the major version here, are there changes outside of this PR going in? Can't see any breaking changes for which there should be a breaking change here.

Reply:
This will also require version bumps in the `AssemblyInfo.cs`